src/components/colorButton.tsx

```diff
@@ -9,7 +9,7 @@ interface OwnProps {
 const ColorButton: FC<OwnProps> = ({ color, clickHandler, active }) => (
   <div>
     <button
-      aria-label="save color"
```

Comment:
We don't need an `aria-label`, since this is clearly an extension that assists people who can see, and this label would only benefit screen reader users.